examples/categorized-menu/app.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
```

Comment:
Hi @CMTegner, I'm trying to think of a way to rewrite the Custom Menu example for this to help this PR along (it's functionality I need for my project as well) and struggling.

The Custom Menu example creates extra elements in the DOM tree based on a condition (the headers do not exist in the list of `items`), where the Categorized Menu example has the categories as part of the returned list from the server and thus as part of the `items` list.

That core difference sets them apart in my mind, even if the end result (from a UX standpoint) remains the same: unselectable headers.